data/mods/Magiclysm/items/alchemy_items.json

```
@@ -301,5 +301,125 @@
     "bashing": 10,
     "to_hit": 1,
     "category": "spare_parts"
+  },
+  {
+    "id": "alchemist_bomb",
+    "type": "TOOL",
+    "category": "weapons",
+    "name": { "str": "Мana bomb" },
+    "description": "Mana bombs are powerful explosive devices developed by a little-known alchemist group for use against enemy spellcasters.  These bombs are fueled by pure arcane energy and can kill instantly.  The exact workings are unclear, but they appear to use some kind of massive mana burn.",
+    "weight": "430 g",
+    "volume": "200 ml",
+    "price": 0,
+    "price_postapoc": 0,
+    "to_hit": -1,
+    "bashing": 14,
+    "material": [ "steel", "plastic" ],
+    "symbol": "*",
+    "color": "green",
+    "use_action": {
+      "need_wielding": true,
+      "target": "alchemist_bomb_act",
+      "msg": "You mix the contents of the bottle, and it starts to sizzle.",
+      "target_charges": 8,
+      "active": true,
+      "menu_text": "Mix liquid",
+      "type": "transform"
+    },
+    "flags": [ "BOMB", "GRENADE" ]
+  },
+  {
+    "id": "alchemist_bomb_act",
+    "type": "TOOL",
+    "category": "weapons",
+    "name": { "str": "mana bomb (lit)", "str_pl": "mana bombs (lit)" },
+    "description": "The liquid in the bottle is sizzling.  Throwing it away from you, preferably at an enemy, would be a very good idea.",
+    "weight": "430 g",
+    "volume": "200 ml",
+    "price": 0,
+    "price_postapoc": 0,
+    "to_hit": -3,
+    "bashing": 14,
+    "material": [ "paper", "powder" ],
+    "symbol": "*",
+    "looks_like": "fragment_bomb_act",
+    "color": "red",
+    "explode_in_fire": true,
+    "explosion": { "power": 200 },
+    "initial_charges": 8,
+    "max_charges": 8,
+    "turns_per_charge": 1,
+    "use_action": {
+      "type": "explosion",
+      "sound_volume": 0,
+      "sound_msg": "ssss…",
+      "no_deactivate_msg": "The reaction has already started - all you can do is get it away from you!",
+      "explosion": { "power": 200 }
+    },
+    "flags": [ "TRADER_AVOID" ]
+  },
+  {
+    "id": "alchemist_mine",
+    "type": "TOOL",
+    "name": { "str": "alchemist_mine" },
+    "description": "Pair of alchemist bombs with concentrated mana wire, reaction is stable, but any movement in mana wire zone provoke extremely powerful explosion in small area.",
+    "weight": "860 g",
+    "volume": "400 ml",
+    "to_hit": -1,
+    "bashing": 5,
+    "material": [ "steel" ],
```

```suggestion
    "material": [ "steel", "concentrated_mana" ],
```

---

Maybe delete steel at all is a good idea, isnt it?

---

Like, its literally grenades out of thin air, it strange it has steel/paper/plastic.
(ive just copypasted this part from grenades and mines and forget to change this stuff

---

How about crystallized_mana + concentrated_mana?
```suggestion
    "material": [ "crystallized_mana", "concentrated_mana" ],
```
The crystallized mana would be a shell around the concentrated mana.